http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -354,33 +384,46 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       token <- F.fromOption(tokenOpt, AppCreationException(s"Pet not found for user ${userEmail}", Some(ctx.traceId)))
       authHeader = Authorization(Credentials.Token(AuthScheme.Bearer, token))
 
-      op = config.coaAppConfig.coaServices
+      services = appType match {
+        case AppType.Wds => config.wdsAppConfig.coaServices
```

Comment:
possibly I should rename this to `wdsService`? So this would read, `set(config.wdsAppConfig.wdsService)`. The logic pattern could also be entirely changed, if we went with a separate case for WDS appType. I like that this shares the wds getStatus call for both cromwell and wds app types, since it is the same status call.